diff --git a/tmp/unknown-c1-m4-publish-20260911/baseline-urp-settings.asset b/tmp/unknown-c1-m4-publish-20260911/unity/Unknown/Assets/UniversalRenderPipelineGlobalSettings.asset
index b016e2b..8abe5af 100644
--- a/tmp/unknown-c1-m4-publish-20260911/baseline-urp-settings.asset
+++ b/tmp/unknown-c1-m4-publish-20260911/unity/Unknown/Assets/UniversalRenderPipelineGlobalSettings.asset
@@ -68,20 +68,7 @@ MonoBehaviour:
       - rid: 4578582156522029089
       - rid: 4578582156522029090
     m_RuntimeSettings:
-      m_List:
-      - rid: 4578582156522029057
-      - rid: 4578582156522029059
-      - rid: 4578582156522029060
-      - rid: 4578582156522029061
-      - rid: 4578582156522029067
-      - rid: 4578582156522029068
-      - rid: 4578582156522029069
-      - rid: 4578582156522029073
-      - rid: 4578582156522029078
-      - rid: 4578582156522029083
-      - rid: 4578582156522029085
-      - rid: 4578582156522029086
-      - rid: 4578582156522029087
+      m_List: []
   m_AssetVersion: 10
   m_ObsoleteDefaultVolumeProfile: {fileID: 0}
   m_RenderingLayerNames:
